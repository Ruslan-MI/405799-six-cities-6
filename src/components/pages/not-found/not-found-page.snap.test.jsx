import React from "react";
import {
  render,
} from "@testing-library/react";
import NotFoundPage from "./not-found-page";
import {
  StoreNameSpace,
} from "../../../const";
import Test, {
  mockUserInitialState,
} from "../../../mocks-for-test";

const dataForStore = {
  [StoreNameSpace.USER]: mockUserInitialState,
};

it(`"Not found" page should render correctly`, () => {
  const {
    container,
  } = render(<Test dataForStore={dataForStore}>
    <NotFoundPage />
  </Test>);

  expect(container).toMatchSnapshot();
});
