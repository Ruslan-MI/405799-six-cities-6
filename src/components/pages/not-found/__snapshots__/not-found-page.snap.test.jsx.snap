// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"Not found" page should render correctly 1`] = `
<div>
  <div
    class="page page--gray page--not-found"
  >
    <header
      class="header"
    >
      <div
        class="container"
      >
        <div
          class="header__wrapper"
        >
          <div
            class="header__left"
          >
            <a
              class="header__logo-link"
              href="/"
            >
              <img
                alt="6 cities logo"
                class="header__logo"
                height="41"
                src="img/logo.svg"
                width="81"
              />
            </a>
          </div>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
              style="flex-direction: column; align-items: flex-end;"
            >
              <li
                class="header__nav-item user"
              >
                <a
                  class="header__nav-link header__nav-link--profile"
                  href="/login"
                >
                  <div
                    class="header__avatar-wrapper user__avatar-wrapper"
                  />
                  <span
                    class="header__login"
                  >
                    Sign in
                  </span>
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </header>
    <main
      class="page__main page__main--not-found"
    >
      <h1
        style="text-align: center;"
      >
        404 Not Found
      </h1>
    </main>
  </div>
</div>
`;
